SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -231,12 +231,10 @@ void USpatialWorkerConnection::OnPlayerIdentityToken(void* UserData, const Worke
 void USpatialWorkerConnection::StartDevelopmentAuth(FString DevAuthToken)
 {
 	Worker_Alpha_PlayerIdentityTokenRequest PITParams{};
-	FTCHARToUTF8 DAToken(*DevAuthToken);
-	FTCHARToUTF8 PlayerId(*SpatialConstants::DEVELOPMENT_AUTH_PLAYER_ID);
-	PITParams.development_authentication_token = DAToken.Get();
-	PITParams.player_id = PlayerId.Get();
-	PITParams.display_name = "";
-	PITParams.metadata = "";
+	PITParams.development_authentication_token = TCHAR_TO_UTF8(*DevAuthToken);
```

Comment:
TCHAR_TO_UTF8 uses an internal temporary and can't be used like this. It's a pain. You need to use the FTCHARToUTF8 struct similar to the flow before.